aspect/build_dependencies.bzl

```diff
@@ -409,19 +409,21 @@ def _target_within_project_scope(label, include, exclude):
     if repo != "":
         return False # We don't support external includes, so all external repos are outside the project scope
     if include:
-        for inc in include.split(","):
-            # This is not a valid label, but can be passed to aspect when `directories: .` is set in the projectview
-            if (inc == "//"):
-                result = True
-                break
-
-            inc = Label(inc)
-            if _get_repo_name(inc) == repo and _package_prefix_match(package, inc.package):
-                result = True
-                break
```

Comment:
I personally prefer this implementation. But of course I'm biased here. I think we should either keep the original implementation or use the AOSP implementation.

Reply:
I think the AOSP implementation is better, because it doesn't try to use a comma-delimited string as a makeshift collection.